usr/src/cmd/smbsrv/smbadm/smbadm.c

```diff
@@ -1408,11 +1408,12 @@ smbadm_group_add_del_member(char *gname, char *mname,
 		 *
 		 * We'e like the real account type if we can get it,
```

Comment:
Spelling error? Although not something this PR introduces.

Reply:
"We'e" should be "We're"

Reply:
Or perhaps _We'd_?

Reply:
I assumed `We'e` had to be `We'd` like @jclulow said. Or maybe just `We`

Reply:
Ah, I see.  Thanks.